packages/block-library/src/image/utils.js

```diff
@@ -3,6 +3,15 @@
  */
 import { NEW_TAB_REL } from './constants';
 
+export function evalAspectRatio( value ) {
```

Comment:
I guess I chose `eval` because it's doing the same thing as the JS `eval()` taking some source code string (in our case a CSS aspect-ratio value) and returning the numerical aspect ratio. `eval('5/8')`, for example, does the same thing.

I'd expect something with `calculate` in the name to take a numerical `width` and `height` argument instead of a single string.